projects/politie/ngx-sherlock/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "@politie/ngx-sherlock",
+    "version": "2.0.0-alpha.0",
+    "peerDependencies": {
+        "@angular/common": "^6.0.0-rc.0 || ^6.0.0",
+        "@angular/core": "^6.0.0-rc.0 || ^6.0.0",
```

Comment:
Angular 7 is out RIGHT NOW. I don't see anything that should prevent this from working with angular 7, but these peer dependencies will nag the heck out of people.